.github/workflows/ci.yaml

```diff
@@ -137,6 +137,7 @@ jobs:
       TEST_DATABASE_MYSQL: "mysql://root:test@(localhost:3306)/mysql?parseTime=true&multiStatements=true"
       TEST_DATABASE_COCKROACHDB: "cockroach://root@localhost:26257/defaultdb?sslmode=disable"
     strategy:
+      fail-fast: false
```

Comment:
It's pretty annoying that the other runs are cancelled if one failed, that way the logs don't contain the end summary, and it's hard to tell if the issue is DB specific or happens on the other runs as well.